sdk/formrecognizer/ai-form-recognizer/samples/javascript/recognizeContent.js

```diff
@@ -14,11 +14,23 @@ const path = require("path");
 const dotenv = require("dotenv");
 dotenv.config();
 
+/**
+ * Make a string representing a bounding box.
+ */
+function boundingBoxToString(box) {
```

Comment:
does JS has like a ToString equivalent? where this might be worth adding to the library?

Reply:
I did consider that, because I'm not really happy with this box -> string helper method, but I chose to go with it because our bounding box type isn't a class. If it was a class, then I would happily just implement `toString`, but since it is an Array, I don't really want to override the default behavior of `toString` that comes from Array. It is worth considering, though I think it is probably a niche case. Most people interested in getting a text representation of it will use `JSON.serialize`.